<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>graph_window</class>
 <widget class="QDialog" name="graph_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="5,1,3">
   <item>
    <widget class="QCustomPlot" name="widget" native="true"/>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout_4">
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_5"/>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Plot</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="3">
      <widget class="QLineEdit" name="lineEdit_2"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit_3"/>
     </item>
     <item row="4" column="3">
      <widget class="QLineEdit" name="lineEdit_4"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>X_MIN</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>X_MAX</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Y_MAX</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Y_MIN</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
